packages/discovery-provider/plugins/pedalboard/apps/app-template/src/index.ts

```diff
@@ -2,13 +2,16 @@ import { log } from '@pedalboard/logger'
 import App from '@pedalboard/basekit/src/app'
 import moment from 'moment'
 
-type SharedData = {}
+type SharedData = object
 
 const main = async () => {
   await new App<SharedData>({})
-    .tick({ seconds: 5 }, async (_app) => {
-      console.log(`tick ${moment().calendar()}`)
-    })
+    .tick({ seconds: 5 }, async () =>
+      // appData
```

Comment:
comment?

Reply:
This is intentional because it's the template for new pedalboard plugins. But the linter was complaining because the variable was unused. So I commented it out instead